SUPPORT.md

```diff
@@ -0,0 +1,21 @@
+# **Support**
+
+Welcome to our project! We're here to help you with any questions or issues you may encounter while using our software. Here are some ways to get in touch with us:
+
+## **Issue Tracker**
+
+If you find a bug or have a feature request, please create an issue in our **[issue tracker](https://github.com/yourusername/yourproject/issues)**. When creating an issue, please provide as much detail as possible about the problem or feature request. This will help us address the issue or request as quickly and efficiently as possible.
+
+## **Email Support**
+
+If you have a question or issue that you don't want to post publicly in the issue tracker, you can email us at [zhangqi444@gmail.com](mailto:zhangqi444@gmail.com) or [lundechen@shu.edu.cn](mailto:lundechen@shu.edu.cn). We will do our best to respond to your email as soon as possible.
+
+## **Community Support**
+
+We also have a **[community forum](https://forum.yourproject.com/)** where you can ask questions and get help from other users of our software. This can be a great resource for getting help with common issues or learning about new features.
+
+## **Social Media**
+
+You can also follow us on **[Twitter](https://twitter.com/yourproject)** or **[Facebook](https://facebook.com/yourproject)** to stay up to date with the latest news and updates about our software.
+
+Thank you for using our project! We're here to help you get the most out of our software, so please don't hesitate to get in touch if you need help.
```

Comment:
Change the url placeholder to `https://TODO`

Reply:
Thanks for the comment, I will modify it.